### PR TITLE
Add structlog.stdlib.recreate_defaults

### PR DESCRIPTION
**Pull request by hynek**
@pradyunsg you asked for this, you're reviewing it. ;)

### CODE REVIEW

**Review by pradyunsg — COMMENTED** [review]
LGTM, based on a quick skim. I can take a closer look over the weekend!

File: src/structlog/_config.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 .. note::
 
    Any changes to these defaults must be reflected in `getting-started`.
+   Also structlog.stdlib.recreate_defaults().
```
[the remark this answers is not in the document]
Comment (hynek) [reply src/structlog/_config.py:36]: These aren't docs even tho they look like it – it's just a reminder to update those places and it shouldn't be rendered anywhere. The correct move would probably be to remove the `.. note::` directive.

[the remark this answers is not in the document]
Comment (hynek) [reply src/structlog/_config.py]: fixed in cc2234c

**Comment by hynek** [comment]
> LGTM, based on a quick skim.

Thanks!

>  I can take a closer look over the weekend!

Naw, this baby is overdue. 😅

**Pull request merged**